### ISSUE TITLE
Work for Regression

### ISSUE THREAD

**Issue by keyplay** [smonsays/equilibrium-propagation#1]
May I know if this model can work for regression? I tried removing softmax before output and used mse loss. The loss decreased but predicted value is far from the true value.

**Comment by smonsays** [comment]
Yes, in principle but you will have to retune the hyperparameters (learning steps, learning rate, ..) accordingly

**Comment by keyplay** [comment]
Thanks for your reply. I use the energy of last layer as the regression output. But the result looks like not correct. Why do I need to return hyper parameter?
